From 325f778f3bbb1b13c436e7754dd22097d31c04d2 Mon Sep 17 00:00:00 2001
From: Waldemar Brodkorb <wbx@uclibc-ng.org>
Date: Thu, 21 Aug 2014 22:02:26 +0200
Subject: [PATCH 07/11] disable test for non-MMU systems

This test fails for non-MMU systems, because it uses fork()

Signed-off-by: Waldemar Brodkorb <wbx@uclibc-ng.org>
---
 test/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/Makefile b/test/Makefile
index 787c530..d51bb62 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -42,6 +42,10 @@ endif
 ifeq ($(HAS_NO_THREADS),y)
 	DIRS := $(filter-out pthread,$(DIRS))
 endif
+# librt test needs fork()
+ifneq ($(ARCH_USE_MMU),y)
+	DIRS := $(filter-out librt,$(DIRS))
+endif
 
 test check all: run
 
-- 
1.8.5.2 (Apple Git-48)

